src/generators/cplusplus/CplusplusConstrainer.ts

```diff
@@ -0,0 +1,71 @@
+import { TypeMapping } from '../../helpers';
+import {
+  defaultEnumKeyConstraints,
+  defaultEnumValueConstraints
+} from './constrainer/EnumConstrainer';
+import { defaultModelNameConstraints } from './constrainer/ModelNameConstrainer';
+import { defaultPropertyKeyConstraints } from './constrainer/PropertyKeyConstrainer';
+import { CplusplusDependencyManager } from './CplusplusDependencyManager';
+import { CplusplusOptions } from './CplusplusGenerator';
+
+export const CplusplusDefaultTypeMapping: TypeMapping<
+  CplusplusOptions,
+  CplusplusDependencyManager
+> = {
+  Object({ constrainedModel }): string {
+    //Returning name here because all object models have been split out
+    return `${constrainedModel.name}*`;
+  },
+  Reference({ constrainedModel }): string {
+    return `${constrainedModel.name}*`;
+  },
+  Any(): string {
+    return 'void*';
```

Comment:
I recently learned about [std::any](https://en.cppreference.com/w/cpp/utility/any), let's use that here instead of `void*`. It needs a `'#include <utility>'` dependency.

Reply:
Okay 👍